src/python/pants/backend/jvm/tasks/coursier_resolve.py

```diff
@@ -54,7 +49,7 @@ def subsystem_dependencies(cls):
   @classmethod
   def register_options(cls, register):
     super(CoursierMixin, cls).register_options(register)
-    register('--allow-global-excludes', type=bool, advanced=False, fingerprint=True, default=False,
+    register('--allow-global-excludes', type=bool, advanced=False, fingerprint=True, default=True,
```

Comment:
This is analogous to `--soft-excludes` which was default to `False`, which is the opposite of this, so correcting it here. The change does not cross release boundary, hence skipping deprecation step.